airflow/api_fastapi/core_api/routes/public/task_instances.py

```diff
@@ -335,3 +337,40 @@ def get_task_instances(
         ],
         total_entries=total_entries,
     )
+
+
+@task_instances_router.get(
+    "/{task_id}/tries/{task_try_number}",
+    responses=create_openapi_http_exception_doc(
+        [status.HTTP_401_UNAUTHORIZED, status.HTTP_403_FORBIDDEN, status.HTTP_404_NOT_FOUND]
```

Comment:
```suggestion
        [status.HTTP_404_NOT_FOUND]
```

Reply:
we don't need to add 401, 403 separately. They are now automatically added

Reply:
oh, got it